packages/flutter_driver/lib/src/driver/timeline_summary.dart

```diff
@@ -102,6 +105,27 @@ class TimelineSummary {
       .where((Duration duration) => duration > kBuildBudget)
       .length;
 
+  /// Average amount of time spent per frame in the engine overall.
+  ///
+  /// Throws a [StateError] if this summary contains no timeline events.
+  double computeAverageFrameOverallTimeMillis() {
```

Comment:
I'm not sure that I understand what this is measuring? what is "time spent in the engine overall?".

Reply:
Quite similar to `FrameTiming.rasterFinish - rasterFinish.buildStart` (but here we are purely extracting the information from timeline data instead of FrameTiming data). I can just rename it